src/reducer.ts

```diff
@@ -18,6 +18,7 @@ export function resourceReducer(
                 ...state,
                 [resourceType]: {
                     isBusy: true,
+                    isSuccess:false,
```

Comment:
Is this really neccessary ? Because in terms of meaning, `_REQUESTED` and `_INIT` states are not actually failing anything. So, setting a property to `false` is not actually reflect the behaviour and might cause confusions and or side effects.

I'd suggest either removing this state or making the type of `isSuccess` `boolean?` instead of `boolean`.

Reply:
Now the isSuccess can be undefined as how it should be.